util/tooltool/tooltool.py

```diff
@@ -41,8 +41,8 @@
 import tempfile
 import threading
 import time
-import urllib2
-import urlparse
+import urllib2  # pylint: disable=import-error
+import urlparse  # pylint: disable=import-error
```

Comment:
Yeah, this tooltool should go away :(

FuzzManager & fuzzfetch are now using requests instead of trying for py2/3 urllib compatibility.